modules/secure-cloud-run-core/main.tf

```diff
@@ -0,0 +1,40 @@
+/**
+ * Copyright 2022 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+module "cloud_run" {
+  source  = "GoogleCloudPlatform/cloud-run/google"
```

Comment:
This should point to the local module like https://github.com/terraform-google-modules/terraform-google-kubernetes-engine/blob/23228a82077646d9019e09df02000e00b74013e4/modules/safer-cluster/main.tf#L22